tensorboard/webapp/metrics/views/card_renderer/scalar_card_line_chart_module.ts

```diff
@@ -14,14 +14,19 @@ limitations under the License.
 ==============================================================================*/
 import {CommonModule} from '@angular/common';
 import {NgModule} from '@angular/core';
+import {CardFobModule} from '../../../widgets/card_fob/card_fob_module';
 import {LineChartModule as LineChartV2Module} from '../../../widgets/line_chart_v2/line_chart_module';
 import {ScalarCardLineChartComponent} from './scalar_card_line_chart_component';
 import {ScalarCardLineChartContainer} from './scalar_card_line_chart_container';
-import {ScalarCardModule} from './scalar_card_module';
+import {ScalarCardFobController} from './scalar_card_fob_controller';
 
 @NgModule({
-  declarations: [ScalarCardLineChartContainer, ScalarCardLineChartComponent],
-  exports: [ScalarCardLineChartContainer],
-  imports: [CommonModule, LineChartV2Module, ScalarCardModule],
+  declarations: [
+    ScalarCardLineChartContainer,
+    ScalarCardLineChartComponent,
+    ScalarCardFobController,
+  ],
+  exports: [ScalarCardLineChartContainer, ScalarCardFobController],
```

Comment:
Could you put a TODO here to remove the ScalarCardFobController export once ScalarCardLineChartModule replaces line chart in scalar card?